### ISSUE TITLE
Copy image in polacode

### ISSUE THREAD

**Issue by Kevinwuu** [octref/polacode#132]
When i need to make more than 1 screenshot, i think saving file for every picture it's kind of unnecessary.
If i can just copy and paste the image to others and make the next one immediately, it will be great. 👍 
I used to use [Pickpic](https://picpick.app/en/) to make screenshot and it can do the same thing as i said.

**Comment by Ranguna** [comment]
Looks like this repo hasn't been active for a while. I want this feature too so I'll be switching to https://marketplace.visualstudio.com/items?itemName=adpyke.codesnap

**Comment by MillhioreBT** [comment]
I agree with @Kevinwuu, I don't know how to compile and install changes on this extension, I would like to try if someone helps me.

Use: 
```js
const { copyImg } = require('img-clipboard');
 
copyImg(Buffer.from(imageData, 'base64'));
```

Example:
```js
const writeSerializedBlobToFile = (serializeBlob, fileName) => {
  const bytes = new Uint8Array(serializeBlob.split(','))
  fs.writeFileSync(fileName, Buffer.from(bytes))
  copyImg(Buffer.from(bytes)) // <HERE>
}
```
this functionality can be achieved, pls if anyone knows how to install these changes help me.

**Comment by djD-REK** [comment]
> Looks like this repo hasn't been active for a while. I want this feature too so I'll be switching to https://marketplace.visualstudio.com/items?itemName=adpyke.codesnap

Thanks for the tip! https://github.com/kufii/CodeSnap
